_data/docs.yml

```
@@ -39,20 +39,30 @@
 
 - title: Basic Tutorials
   docs:
-   - tutorials/basic/getting_started
+   - tutorials/core/getting_started
    - tutorials/basic/getting_started_embedded
```

```suggestion
   - tutorials/core/getting_started_embedded
```
We should rename the folder `basic` completely to `core`.

---

Done